R/AbstractPackageReporter.R

```diff
@@ -171,8 +171,8 @@ AbstractPackageReporter <- R6::R6Class(
             )
 
             if (! pkg_name %in% installed_packages){
-                msg <- sprintf("pkgnet could not find a package called '%s'.", pkg_name)
-                log_fatal(msg)
+              msg <- sprintf("pkgnet could not find a installed package named '%s'. Please install the package first.", pkg_name)
```

Comment:
thanks! can you please change `a installed` to `an installed`

Reply:
Sure, will do.

Reply:
Changed that as well as the error message check in `tests/testthat/test-AbstractPackageReporter.R`. Please have a look.